packages/adminrouter/extra/src/nginx.master.conf

```diff
@@ -368,6 +349,7 @@ http {
                 auth.access_mesosdns_endpoint();
             }
             include common/proxy-headers.conf;
+            proxy_set_header Authorization "";
```

Comment:
I don't feel strong about separating single directive into a file, it was mostly just an idea. I am ok if it stays as it is.